.gitignore

```diff
@@ -100,3 +100,9 @@ packages/**/playwright-report/index.html
 
 .env
 dist
+<<<<<<< HEAD
+
+# playwright non-essentials
+packages/gui/playwright-report
+=======
+>>>>>>> main
```

Comment:
Looks like there are some git merge conflict markers left in this file.

Since we already have `packages/**/playwright-report/index.html` on line 99, I think we can just delete lines 103-108